### DISCUSSION TITLE
快速跳转目录  能否增加支持Dopus TC XYplorer等

### DISCUSSION THREAD

**Discussion by bytimer** [shuax/MouseInc.Settings#393] [Ideas]
listary快速跳转目录 支持的文件管理器有Directory Oopus | Total Commander | XYplorer | Xplorer2 | FreeCommander | 资源管理器
希望MouseInc 也能支持多一些第三方文件管理器（估计要花点功夫）

这里有AHK脚本实现DOpus的跳转
https://github.com/Chaoses-Ib/IbDOpusScripts/blob/main/Scripts/DialogJump.ahk   
这里也是个AHK脚本  支持的文件管理器更多
https://www.voidtools.com/forum/viewtopic.php?f=2&t=11194
可以参考一下如何获取第三方文件管理器的当前路径

我是主用DOpus的，TC用的人也很多；  这两个是最强大的
如果大神您时间不够的话，希望能优先支持DO和TC吧
感谢您的软件！

**Comment by shuax** [comment]
可以参考一下
